Tools/Scripts/libraries/webkitscmpy/setup.py

```diff
@@ -29,7 +29,7 @@ def readme():
 
 setup(
     name='webkitscmpy',
-    version='5.6.1',
+    version='5.6.2',
```

Comment:
Need to modify `Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py` too